components/post_view/message_attachments/message_attachment/__snapshots__/message_attachment.test.tsx.snap

```diff
@@ -25,6 +25,7 @@ exports[`components/post_view/MessageAttachment should call actions.doPostAction
     >
       <a
         href="author_link"
+        key="attachment__author-name"
```

Comment:
Quite a few of these snapshots are changing. Is that intentional?

Reply:
The changes are indeed intentional. I will comment on each one.
These here are due to a simplification of the typescript code that include this in a list, and therefore it needs a key.